src/filtrering/filtrering-mine-filter/mine-filter-rad.tsx

```diff
@@ -1,57 +1,58 @@
-import {MineFilter} from '../../ducks/mine-filter';
 import {useDispatch, useSelector} from 'react-redux';
 import {AppState} from '../../reducer';
 import {finnSideNavn, mapVeilederIdentTilNonsens} from '../../middleware/metrics-middleware';
 import {logEvent} from '../../utils/frontend-logger';
 import {velgLagretFilter} from '../../ducks/filtrering';
-import {apneMineFilterModal} from '../../ducks/mine-filter-ui';
+import {apneLagreFilterModal, markerValgtLagretFilter} from '../../ducks/lagret-filter-ui-state';
 import {Radio} from 'nav-frontend-skjema';
 import RedigerKnapp from '../../components/knapper/rediger-knapp';
 import React, {RefObject, useRef} from 'react';
 import {antallFilter} from '../../components/modal/mine-filter/mine-filter-utils';
 import {ListevisningType} from '../../ducks/ui/listevisning';
+import {LagretFilter} from "../../ducks/lagretFilter";
 
 interface LagretFilterRadProps {
-    filter: MineFilter;
-    filtergruppe: string;
+    lagretFilter: LagretFilter;
+    filtergruppe: ListevisningType;
     parentDiv: RefObject<HTMLDivElement>;
 }
 
-function MineFilterRad({filter, filtergruppe, parentDiv}: LagretFilterRadProps) {
+function MineFilterRad({lagretFilter, filtergruppe, parentDiv}: LagretFilterRadProps) {
     const dispatch = useDispatch();
     const checkboxRef = useRef<HTMLDivElement>(null);
 
     const valgtLagretFilter = useSelector((state: AppState) =>
         filtergruppe === ListevisningType.minOversikt
-            ? state.mineFilterMinOversikt.valgtMineFilter
-            : state.mineFilterEnhetensOversikt.valgtMineFilter
+            ? state.mineFilterMinOversikt.valgtLagretFilter
+            : state.mineFilterEnhetensOversikt.valgtLagretFilter
     );
     const veilederIdent = useSelector((state: AppState) => state.inloggetVeileder.data!);
     const veilederIdentTilNonsens = mapVeilederIdentTilNonsens(veilederIdent.ident);
 
     function velgFilter() {
         logEvent(
             'portefolje.metrikker.lagredefilter.valgt-lagret-filter',
-            {antallFilter: antallFilter(filter.filterValg)},
+            {antallFilter: antallFilter(lagretFilter.filterValg)},
             {
-                filterId: filter.filterId,
+                filterId: lagretFilter.filterId,
                 sideNavn: finnSideNavn(),
                 id: veilederIdentTilNonsens
             }
         );
-        dispatch(velgLagretFilter(filter, filtergruppe));
+        dispatch(velgLagretFilter(lagretFilter, filtergruppe));
+        dispatch(markerValgtLagretFilter(lagretFilter, filtergruppe));
     }
 
     function onClickRedigerKnapp() {
-        dispatch(apneMineFilterModal(filtergruppe));
+        dispatch(apneLagreFilterModal(filtergruppe));
```

Comment:
change back to apneMineFilterModal (sorry!!)

Reply:
Its ok, fixed :)